Lib/test/test_genericalias.py

```diff
@@ -1,5 +1,6 @@
 """Tests for C-implemented GenericAlias."""
 
+import collections
```

Comment:
This file does `from collections.abc import *` on lines 10, so I'm not sure we need this.

Reply:
Ah, right, I'd missed the `from collections.abc import *`. Fixed.

Reply:
By the way, you've introduced me today to yet another quirk of Python's import system that I had no idea was a thing:

```python
>>> import collections
>>> collections.abc.Iterable
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'collections' has no attribute 'abc'
>>> from collections.abc import *
>>> collections.abc.Iterable
<class 'collections.abc.Iterable'>
```